streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/ScalaSerde.scala

```diff
@@ -1,70 +0,0 @@
-/*
- * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
- * Copyright (C) 2017-2018 Alexis Seigneurin.
- *
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements. See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License. You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.kafka.streams.scala
-
-import org.apache.kafka.common.serialization.{Serde, Deserializer => JDeserializer, Serializer => JSerializer}
-
-trait ScalaSerde[T] extends Serde[T] {
```

Comment:
Since we are moving around these classes, we'd also need to double check if any of the corresponding web docs in `docs/streams` need to be updated. I'd suggest making a pass over the doc changes in https://github.com/apache/kafka/pull/4756 and see if any of them need to be updated accordingly.

For example, if we have in any places mentions about the wrapper serializer / deserializer / etc, we should remove them then.

Reply:
I was thinking about this indeed. I will have a check this evening and push any changes.